pkg/cloud/services/networking/securitygroups.go

```diff
@@ -25,12 +25,15 @@ import (
 	"sigs.k8s.io/cluster-api-provider-openstack/pkg/record"
 )
 
+// Exported Constants can be used to calculate securitygroupnames in other packages
```

Comment:
This PR focus on fixing the bug, not update variable name. If you want to refactor, raise another PR, please.